be/src/exec/vectorized/json_scanner.h

```diff
@@ -101,12 +101,15 @@ class JsonReader {
     bool _strict_mode = false;
 
     std::shared_ptr<SequentialFile> _file;
-    int _next_line;
-    int _total_lines;
     bool _closed;
     std::vector<SlotDescriptor*> _slot_descs;
 
-    std::unique_ptr<uint8_t[]> _json_binary_ptr;
+    // For performance reason, the simdjson parser should be reused over several files.
+    //https://github.com/simdjson/simdjson/blob/master/doc/performance.md
+    simdjson::ondemand::parser _simdjson_parser;
+    std::unique_ptr<uint8_t[]> _parser_buf;
```

Comment:
Will every load have a _parse_buf?
Does it will consumes much memory for a high concurrency load?

Reply:
The previous implement is one buffer for one message. The new implement is one buffer for one load, which would not bring extra memory consuming.

Reply:
How much memory will a load consume?